core/src/conn.rs

```diff
@@ -146,15 +145,15 @@ impl MongoConnection {
 
         let command_response = libmongosqltranslate_run_command(command)?;
 
-        if let CommandResponse::GetMongosqlTranslateVersion(response) = command_response {
-            Ok(response.version)
+        if let CommandResponse::GetMongosqlTranslateVersion(version) = command_response {
+            Ok(version.version)
         } else {
             unreachable!()
         }
     }
 
     fn is_libmongosqltranslate_compatible_with_driver_version() -> Result<bool> {
-        let command = CheckDriverVersion::new(DRIVER_ODBC_VERSION.clone());
+        let command = CheckDriverVersion::new(DRIVER_METRICS_VERSION.clone());
```

Comment:
This is valid semver, the other is not.

Reply:
Yeah it was locked on Oliver's PR and there was nothing I could do but delete the repo locally to fix this. I'll update the ticket.

Reply:
Oh yeah, I was pointing to `master` branch at first incorrectly until Ethan pointed it out.  I updated it Friday afternoon.

Reply:
Ah this may have been the root cause. Git was probably trying to protect me, or I put it into an internal state of "WTFLOLBBQ on your own buddy"